python/cudf/cudf/tests/test_hdfs.py

```diff
@@ -0,0 +1,163 @@
+import os
+from io import BytesIO
+
+import fastavro as fa
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+import pyarrow.orc as orc
+import pytest
+
+import cudf
+from cudf.tests.utils import assert_eq
+
+if not os.environ.get("RUN_HDFS_TESTS"):
```

Comment:
We need CI to test this. So perhaps this env var needs to be set in https://github.com/rapidsai/cudf/blob/branch-0.9/ci/gpu/build.sh?

Reply:
Hi @shwina. Testing this in CI would require a host of changes like having java and Hadoop on the CI and HDFS configured. The initial idea of keeping this environment variable was to have something where HDFS tests could be run by CI if requested using the environment variable. (as and when Hadoop is added to the environment).
If needed I can set the tests to xfail instead of skipping based on the variable (giving the impression that CI can test these)

Reply:
Note: The current approach is somewhat similar to how Dask does the HDFS tests and has a similar environment for testing HDFS. https://github.com/dask/dask/tree/master/continuous_integration/hdfs

Also the tests were run locally in a Cloudera CDH5 Hadoop environment (pseudo distributed)  based on https://www.cloudera.com/documentation/enterprise/5-9-x/topics/cdh_qs_yarn_pseudo.html

Reply:
Makes sense - thanks for the explanation!